src/server/server.js

```diff
@@ -0,0 +1,157 @@
+const express = require("express");
+const app = express();
+const http = require("http").createServer(app);
+const io = require("socket.io")(http);
+const mongoose = require("mongoose");
+const passport = require("./auth.js");
+const session = require("express-session");
+const validator = require("email-validator");
+
+const User = require("./schemas/User.js");
+
+mongoose.connect("mongodb://localhost:27017/kitchen-table-magic");
```

Comment:
Abstract the MONGODB_URI into an environment variable check, because we'll quickly want to support different values here depending on the environment the application is running in. Something like:

`const MONGODB_URI = process.env.MONGODB_URI || 'mongodb://localhost:27017/kitchen-table-magic'`

And then:

`mongoose.connect(MONGODB_URI)`